lib/collections/addon/provider/moderation/styles.scss

```diff
@@ -0,0 +1,22 @@
+@import 'app/styles/layout';
```

Comment:
No change necessary, but in a few years [this will no longer be supported](https://sass-lang.com/documentation/at-rules/import), but instead will be using the `@use` rule. However, only dart-sass supports that now, not node-sass, so for the moment we have to continue to use this.